### PR TITLE
AB facs: label y axis in sketching canvas

### LINKED ISSUE #627: FACS: y axis labeling
Can we find a standard label, like "% of cells" ?

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?

Fix #627 
Depends on #
#### What's this PR do?

Add labels to y axis in the sketching tool.

### CODE REVIEW

File: instructor/ui/sketch_tool.js [hunk 1]
```diff
@@ -18,29 +22,33 @@ function draw_graph_background(x_upper_bound, tick_values){
...
-        text_obj.content = value;
-        // need to subtract half of the width from x
-        text_obj.position.x -= text_obj.bounds.width / 2;
+    var x_coor, y_coor;
+    _.each(tick_values, function(value){
+        x_coor = X_AXIS_LENGTH_PX * value / x_upper_bound + X_ORIGIN;
+        y_coor = Y_ORIGIN + 13;
+        printText(x_coor, y_coor, value);
+    });
+    _.each([20, 40, 60, 80], function(value){
```
Comment (pdpinch) [inline instructor/ui/sketch_tool.js]: Are the y axis tick marks fixed at 20, 40, 60, 80? If they are, we should add a comment here.

Comment (annagav) [inline instructor/ui/sketch_tool.js]: yes, they are fixed.

File: instructor/ui/sketch_tool.js [hunk 2]
```diff
@@ -128,4 +136,22 @@ function draw_line(x1 ,y1 ,x2 ,y2, color){
...
+    text.rotate(270);
+}
+/* Print PointText */
+function printText(x, y, value){
+    var text_obj = new paper.PointText(new paper.Point(x, y));
+    text_obj.content = value;
+    // need to subtract half of the width from x
+    text_obj.position.x -= text_obj.bounds.width / 2;
+    return text_obj;
 }
```
Comment (pdpinch) [inline instructor/ui/sketch_tool.js]: should we add a newline?

**Comment by pdpinch** [comment]
a couple of small comments, but otherwise 👍

**Pull request merged**